Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -157,9 +158,15 @@ function runHelm(helmCli: helmcli, command: string, kubectlCli: kubernetescli, f
 
     const execResult = helmCli.execHelmCommand();
     tl.setVariable('helmExitCode', execResult.code.toString());
-    if (execResult.stdout) {
-        tl.setVariable('helmOutput', execResult.stdout);
+
+    var commandOutputLength = execResult.stdout.length;
```

Comment:
The `length` property of stdout should be accessed after the null check for `execResult.stdout`. You can put the new logic inside the already existing if condition.

Reply:
The `length` property of stdout is now inside condition. Thanks.